assessment.sql

```diff
@@ -5,24 +5,46 @@ CREATE DATABASE "sql-crud-assessment";
 
 -- Question 1. Create a table called `licensees` to hold data in columns surname,given_name,gender,height,weight,dob,eye_color,hair_color.  See `data/licensees.csv` for example data.  Don't forget that all tables should have an id column as a unique identifier for a row.
 
--- your code here
+CREATE TABLE licensees(
+surname CHARACTER VARYING,
+given_name CHARACTER VARYING,
+gender CHARACTER VARYING(1),
+height integer,
+weight integer,
+dob DATE
+eye_color CHARACTER VARYING,
+hair_color CHARACTER VARYING
+)
```

Comment:
Your statement will not run until a semicolon is found. If something other than a semicolon is what the SQL interpreter encounters next, you've got an error on your hands. 4/5